content/md/products-super-power/work-on-a-product.md

```diff
@@ -95,6 +95,76 @@ From the product form:
 
 If you do not see `+` icon, you might not have the appropriate rights to add an option, please refer to [Manage your user roles](/articles/build-your-user-roles.html).
 
+## Work with assets in an asset collection attribute
+
+In the product sheets, you can have some asset collection attributes which look like this:
+
+![asset_collection_attribute](../img/asset_collection_attribute.png)
+
+In the following section, you will discover all the features that are available around this attribute.
+
+::: ee
+The asset collection attribute is only available in the Enterprise Edition.
+:::
+
+### Link assets to your product
+
+You can link assets to your product using this attribute. To do that, follow these steps:
+
+1. Go to your product form
+1. Scroll down towards the asset collection attribute
+1. Click on `Manage assets` to access to the asset picker and the basket
+1. Pick the assets you want to link to your product, and save.
+
+![basket](../img/basket.png)
+
+### Upload assets directly from the asset collection
+
+From the 2.3 version, you can upload assets directly from the Asset collection in the Product Form. You don't need to quit the product form anymore.
+
+1. Click on the new button `Upload assets` in the asset collection, then you will retrieve the mass upload screen.
```

Comment:
"A mass upload screen will be prompted"?

Reply:
I change "retrieve" by "find"

Reply:
I used "retrieve" 3 times so I change the word 3 times ;)